core/src/leader_schedule_utils.rs

```diff
@@ -94,6 +78,22 @@ pub fn tick_height_to_slot(ticks_per_slot: u64, tick_height: u64) -> u64 {
     tick_height / ticks_per_slot
 }
 
+fn sort_stakes(stakes: &mut Vec<(Pubkey, u64)>) {
```

Comment:
is this function changed or just moved?  if just moved, how about not moving it ;)

Reply:
I want my private functions to come after my public functions :(

Reply:
ok, you go girl

Reply:
how about putting them in a separate impl block?

Reply:
hmmm there's no impl blocks in these, and a separate module seems like overkill